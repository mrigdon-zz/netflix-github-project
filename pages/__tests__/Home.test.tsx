import React from "react";
import { createRenderer } from "react-test-renderer/shallow";
import Home from "..";

describe("Home", () => {
  it("renders home", () => {
    const renderer = createRenderer();
    renderer.render(
      <Home
        locale="en"
        homeParams={{ search: "", ascending: false, column: "stars" }}
      />
    );
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
