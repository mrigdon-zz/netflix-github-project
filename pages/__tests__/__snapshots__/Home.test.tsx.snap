// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders home 1`] = `
<Context.Provider
  value={
    Object {
      "params": Object {
        "ascending": false,
        "column": "stars",
        "search": "",
      },
      "setParams": [Function],
    }
  }
>
  <PageLayout
    locale="en"
  >
    <h1>
      <Label
        name="welcome"
      />
    </h1>
    <p
      className="description"
    >
      <Label
        name="searchDescription"
      />
    </p>
    <OrgSearch />
  </PageLayout>
</Context.Provider>
`;
